schemas/project-scratch-def.schema.json

```diff
@@ -1,807 +0,0 @@
-{
```

Comment:
@ntotten @amphro it's ok to move these files to the top level since those who reference them using the direct link to the repo are using the `master` branch. We've now replaced the `master` branch with `main`.